StorageUtility/Scsi.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+<License>
+Copyright 2015 Virtium Technology
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http ://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+</License>
+*/
+
+#include "Scsi.h"
+
+namespace vtStor
+{
+    namespace StorageUtility
+    {
+        namespace Scsi
+        {
+
+            const U32 SCSI_SECTOR_SIZE_IN_BYTES = 512;
+
+            const U8  CDB_REGISTER_SIZE = 16;
```

Comment:
Are you trying to define the max size of a CDB? If so, rename it MAX_CDB_REGISTER_SIZE